Projects/1.Counter/index.css

```diff
@@ -0,0 +1,69 @@
+body{
+    background-color: lightblue;
+}
+.pointer{
+    cursor: pointer;
+}
+.div{
+    position: relative;
+    left: 430px;
+    right: 20px;
+    top: 100px;
+    background-color: rgb(32, 178, 170);
+    width: 350px;
+    height: 400px;
+    border: 4px solid black;
+    padding: 50px;
+    margin: 20px;
+    text-align: center;
+    border-radius: 3%;
+    font-size: 30px;
+    font-family: 'Times New Roman', Times, serif;
+    font-weight: bold;
+}
+.p{
+    font-size: 28px; font-weight: normal;
+}
+.Increase{
```

Comment:
Lets use lowercase for defining class name. For example, in this case, increase.